modules/Matting/AlphaGAN-Matting/AlphaGAN/alphaGAN_eval.py

```diff
@@ -0,0 +1,241 @@
+import torchvision.transforms as transforms
+import numpy as np
+import torch as t
+from model.AlphaGAN import NetG
+from PIL import Image
+import os
+from visualize import Visualizer
+import tqdm
+
+os.environ["CUDA_VISIBLE_DEVICES"] = '2'
+vis = Visualizer('alphaGAN_eval')
+
+transform = transforms.Compose([
+            transforms.ToTensor(),
+            transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+        ])
+
+to_pil = transforms.Compose([
+    transforms.ToPILImage()
+])
+
+MODEL_DIR = '/data1/zzl/model/alphaGAN/new_trainset/netG/netG_40.pth'
+
+
+def padding_img(img):
+
+    img_size = np.shape(img)
+    if len(img_size) == 3:
+        (h, w, c) = img_size
+        w_padding = (int(w/320) + 1) * 320
```

Comment:
Why not `w//320` instead of `int(w/320)`? Does `w//320` causes computational problems? And as far as I can figure out, you are trying to take `ceil` of w/320? Isn't it? Is `numpy` having any pre-defined method/function for that as you have already imported. 
PS - I am just commenting out of curiosity. I know the results will be correct from both the logics but I want to know that are there any issues with computations between the two ways. Thanks for reading.

Reply:
Also, I am fairly new to `OpenCV` so thought of learning by contributing. Please let me know if this is in the coding style of `OpenCV`.

Reply:
In python w//320 gives the same result as int(w/320)